src/core/ngx_connection.c

```diff
@@ -78,6 +78,7 @@ ngx_create_listening(ngx_conf_t *cf, struct sockaddr *sockaddr,
 
```

Comment:
For the commit message: the limit of chars per line is ~72, not 80. (because 'git show' indents it I think)

Also, the links (`[1]`, etc.) are missing, see your lighttpd commit.

Maybe this could be split into 4 commits, not sure what they usually do (check the git history): one for the core, then one for each feature: http, mail, stream.

Then the titles would be: `core|http|mail|stream: add MPTCP support`